pkg/backend/httpstate/policypack.go

```diff
@@ -124,7 +124,7 @@ func (pack *cloudPolicyPack) Publish(
 		return result.FromError(err)
 	}
 
-	analyzer, err := op.PlugCtx.Host.PolicyAnalyzer(tokens.QName(abs), op.PlugCtx.Pwd)
+	analyzer, err := op.PlugCtx.Host.PolicyAnalyzer(tokens.QName(abs), op.PlugCtx.Pwd, nil)
```

Comment:
Nit: This isn't a strongly followed convention in the codebase, so this is "less than optional", but when passing in some random `nil` parameter it might be nice to add a comment like `nil /* options */` to indicate what component is intentionally not being passed.